blocks/rich-text/format.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * External dependencies
+ */
+import { omitBy, map } from 'lodash';
+import { nodeListToReact } from 'dom-react';
+
+/**
+ * WordPress dependencies
+ */
+import { createElement, renderToString } from '@wordpress/element';
+
+/**
+ * Transforms a WP Element to its corresponding HTML string.
+ *
+ * @param {WPElement} value Element.
+ *
+ * @return {string}         HTML.
```

Comment:
Minor: I don't think we try to align `return` and `param` descriptions now that they're separated with newline.